### DISCUSSION TITLE
Github Education pack not active

### DISCUSSION THREAD

**Discussion by mehedee-iub** [community/community#135730] [GitHub Education]
### Hi everyone,

I got Verified on Aug 10, 2024, But The education pack is not active :(

![image](https://github.com/user-attachments/assets/3da0aa15-0cf7-4c94-9d10-bc351eba8f0e)

**Comment by GabriPalmyro** [comment]
Same here!

**Comment by mehedee-iub** [comment]
Today my package is activated :)
